addons/web/static/src/views/fields/relational_utils.xml

```diff
@@ -25,6 +25,7 @@
                             <t t-else="">Remove</t>
                         </button>
                     </t>
+                    <button class="btn btn-link ms-auto" t-on-click="props.openFormView">More Options</button>
```

Comment:
As "openFormView" is an optional prop, shouldn't this button only be displayed if it has a value?